src/app/home/home.component.ts

```diff
@@ -62,15 +61,9 @@ export class HomeComponent implements OnInit, AfterViewInit, OnDestroy {
   ngOnInit() {
     this.getNotification();
     this.onUserUpdate();
-    this.languages = (<any>languages).map(language => {
-      if (language.servedUrl === document.baseURI) {
-        this.currentFlag = language.shortCode;
-        this.currentLang = language.name;
-      }
-      return language;
-    }).filter(lang  => {
-      return lang['active'] !== 'N';
-    });
+    this.currentLanguage = this.languages.find(language => {

```

Comment:
can we also filter out the current language from the language list. As it does not look nice to let user switch to same language.